common/cache/domainCache.go

```diff
@@ -718,12 +734,14 @@ func (entry *DomainCacheEntry) duplicate() *DomainCacheEntry {
 		VisibilityArchivalStatus: entry.config.VisibilityArchivalStatus,
 		VisibilityArchivalURI:    entry.config.VisibilityArchivalURI,
 		BadBinaries:              copyResetBinary(entry.config.BadBinaries),
+		AsyncWorkflowConfig:      entry.config.AsyncWorkflowConfig.DeepCopy(),
+		// Q: Should we set IsolationGroups as well? Othewise domaincache will not be able to detect changes in isolation groups
```

Comment:
@davidporter-id-au `IsolationGroups` is missed here. Was that intentional? If not I can add it

Reply:
Good spotting, I... think that's correct, we should add it

Reply:
adding it in #5679